Add config class (#1)
planetarium/Unithereum: README.md

[solc]: https://github.com/ethereum/solidity
[C# identifier name]: https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names

### Configuration

> You can override default properties by adding codegen.config.json to Unity Project root.

Reviewer: ```suggestion
You may override the default configuration with a `codegen.config.json` file in the Unity project root.
```

Available options are `dotnetPath`, `nsPrefix`, and `outputDir`.

> Ensure `nsPrefix` should follow [C# identifier name],
> and note that `outputDir` path is relative to Unity `Assets/` directory.

Reviewer: ```suggestion
Ensure that `nsPrefix` conforms to the [C# identifier name] constraints.
Also, note that the `outputDir` is relative to the `Assets/` directory of the Unity project..
```
